templates/blog/post_list.html

```diff
@@ -0,0 +1,18 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Post's List:</h1>
+  <ul>
+    {% for post in post_list%}
```

Comment:
There is a minor issue with spacing in the for loop tag. Consider adding a space between 'post_list' and '%}' for better readability.